users/models.py

```diff
@@ -1,29 +1,71 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals, absolute_import
 
+from datetime import datetime
+
 from django.contrib.auth.models import AbstractUser
 from django.core.urlresolvers import reverse
 from django.db import models
+from django.utils import timezone
+
+# Should be seperate but for clarity I'm putting them together to showcase our models
+# from api.models import Trip
 
-from api.models import Trip
 # from django.utils.translation import ugettext_lazy as _
 
 
-class User(AbstractUser):
+class Trip(models.Model):
+    """ Trip model """
 
-    # First Name and Last Name do not cover name patterns
-    # around the globe.
-    photo = models.URLField(blank=True)
-    phone_number = models.BigIntegerField(blank=True, null=True)
-    is_customer = models.BooleanField(max_length=100, default=True)
-    description = models.CharField(blank=True, max_length=1000)
-    interests = models.CharField(blank=True, max_length=500)
+    trip_id = models.BigIntegerField(blank=True, null=True)
+	scenic = models.BooleanField(blank=True, default=False)
+	start_ts = models.DateTimeField(blank=True, null=True, default=timezone.now)
+	status = models.CharField(blank=True, default='unconfirmed', max_length=200)
+    # Either distance or time to one-to-one with driver
+    # We could reward drivers based on distance
+    distance = models.BigIntergerField(blank=True, null=True)
+    driver = ForeignKey(Driver)
```

Comment:
drivers will have a list of trips, so in models.py for users: `driver = models.ForeignKeyField(Trip, related_name='trips')`